tiller/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
If you are changing by kubernetes version, please use `Capabilities` to support multiple api versions.

Reply:
`extensions/v1beta1` is gone, and `v1beta` has been off for a while, so I don't think it is necessary.